api/endpoints/room.md

```diff
@@ -96,22 +115,28 @@ Collection of all Rooms.
     
     [Room Collection][]
 
+## RoomTypes [/rooms/types///{?includeReservable,includeNonReservable,includeOccupiable,includeNonOccupiable}]
+Collection of all RoomTypes.
 
-### Create a Room [POST]
-The following attributes are required to create a Room: `name`, `floor`, and `type`.
++ Model (application/json)
+    JSON representation of the Room Types resource.
 
-+ Request (application/json)
+    + Body
 
-            {
-                "name": "room 101",
-                "floor": {
-                    "id": 16
-                },
-                "type": {
-                    "id": 44
+            [
+                {
+                    "id":21
+                    "name":"Office",
+                    "sortOrder":5
                 }
-            }
-
-+ Response 201
+            ]
 
-    [Room][]
+### List of all RoomTypes [GET]
++ Parameters
+    + includeReservable (optional, boolean, `true`) ... If true, query will include reservable rooms
+    + includeNonReservable (optional, boolean, `false`) ... If true, query will include non-reservable rooms
+    + includeOccupiable (optional, boolean, `true`) ... If true, query will include occupiable rooms
+    + includeNonOccupiable (optional, boolean, `false`) ... If true, query will include non-occupiable rooms
+    
```

Comment:
you can also add `includeParking` here too

Reply:
added